apps/snitch_core/test/data/schema/package_item_test.exs

```diff
@@ -41,7 +46,34 @@ defmodule Snitch.Data.Schema.PackageItemTest do
              } == errors_on(cs)
     end
 
-    test "with invalid quantity, delta" do
+    test "fails for non-existent line_item_id", context do
```

Comment:
Group together the tests for missing foreign keys in one test.

Reply:
By grouping them together it checks only for one foreign key constraint and ignores the other